NuRadioMC/simulation/simulation.py

```diff
@@ -551,6 +554,12 @@ def run(self):
 
                         electric_field = NuRadioReco.framework.electric_field.ElectricField([channel_id], self._det.get_relative_position(self._sim_station.get_id(), channel_id))
                         electric_field.set_frequency_spectrum(np.array([eR, eTheta, ePhi]), 1. / self._dt)
+                        # Trace start time is equal to the interaction time relative to the first
+                        # interaction plus the wave travel time.
+                        if hasattr(self, '_vertex_time'):
+                            trace_start_time = self._vertex_time + T
+                        else:
+                            trace_start_time = T
                         electric_field.set_trace_start_time(T)
```

Comment:
shouldn't this be `electric_field.set_trace_start_time(trace_start_time)` ?

Reply:
I don't know what happened to me while writing this code, honestly...

Reply:
I'll change everything tomorrow morning.

Reply:
The changes have been addressed.